project.yaml

```diff
@@ -94,12 +96,17 @@ dataSources:
           filter:
             topics:
               - UnregisterIndexer(address indexed indexer)
+        - handler: handleSetCommissionRate
```

Comment:
Question: why move the `handleSetCommissionRate` up here? I suppose it is under `staking` contract?

Reply:
set commission rate was moved under indexerRegistry in contracts. 
https://onfinality.atlassian.net/wiki/spaces/SQ/pages/1836089354/Release+v0.11.1-1